nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -34,7 +37,65 @@ when switching between consoles, and so on. An interesting multi-node test is
 [`nfs/simple.nix`](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/nfs/simple.nix).
 It uses two client nodes to test correct locking across server crashes.
 
-There are a few special NixOS configuration options for test VMs:
+## Calling a test {#sec-calling-nixos-tests}
+
+Tests are invoked a bit differently depending on whether the test lives in NixOS or in another project.
+
+### Testing within NixOS {#sec-call-nixos-test-in-nixos}
+
+Test modules can be instantiated into derivations in multiple ways.
```

Comment:
```suggestion
Test modules can be added to derivations in multiple ways.
```

Not sure if "instantiate" is the right word at all.

Reply:
```suggestion
```

Actually this is stating the obvious, because the three variants follow immediately and there is nothing else in this section. We can remove it.